lib/active_merchant/billing/gateways/airwallex.rb

```diff
@@ -154,6 +154,7 @@ def create_payment_intent(money, options = {})
         post[:merchant_order_id] = merchant_order_id(options)
         add_referrer_data(post)
         add_descriptor(post, options)
+        post['payment_method_options'] = { 'card' => { 'risk_control' => { 'three_ds_action' => 'SKIP_3DS' } } } if options[:skip_3ds] && (options[:skip_3ds].to_s.casecmp('true') == 0)
```

Comment:
I think we can simplify the conditional here: `if options[:skip_3ds] && (options[:skip_3ds].to_s.casecmp('true') == 0)`

I say treat it as a `bool` `true` and then we can just implicitly check `if options[:skip_3ds]`

Reply:
Thanks @naashton, this change is updated 👍